scripts/py_matter_idl/matter_idl/matter_grammar.lark

```diff
@@ -78,13 +87,13 @@ bool_default: "true"i  -> bool_default_true
             | "false"i -> bool_default_false
 ?default_value: "default"i "=" (integer | ESCAPED_STRING | bool_default)
 
-constant_entry: id "=" positive_integer ";"
+constant_entry: [maturity] id "=" positive_integer ";"
```

Comment:
I assume this covers enum and bitmap values?

Reply:
yes, both contain constants.